### PR TITLE
Final testing

### PR DESCRIPTION
**Pull request by RyanRio**
This PR proves the backend logic works, however in the tests the last and second to last test of orderSpec cannot be run simultaneously - one has to be temporarily disabled. The issue is somehow when the second to last test is also running, the data sent to admin by the backend becomes null (after the client submits an order request), this is a trivial error because on the adminclient two sockets will not be simultaneously active, however for the sake of testing it should probably be fixed. Thus I will submit this PR and then create an issue.

### CODE REVIEW

**Comment by stardust66** [comment]
@RyanRio I don't think that's true. I think the error comes from there already being an order with id:1. I just changed the expected id value to 2 and all the tests work.

**Comment by stardust66** [comment]
Last commit fixes #17, but we need to find a way to reset the id counter in between tests.

File: spec/auth/orderSpec.js [hunk 1]
```diff
@@ -134,4 +136,56 @@ describe('OrderController', () => {
             });
         });
     });
-});
+
+    it('receives a completion request from the admin and notifies the user that their order is complete', (done) => {
```
Comment (stardust66) [inline spec/auth/orderSpec.js]: Good test, but I think we need to clean up the callbacks a little. This could be done later, as it's not that important.

Comment (RyanRio) [reply spec/auth/orderSpec.js]: Personally I think the callbacks are logical/as nice as they get.

**Comment by stardust66** [comment]
This completes #11.

**Pull request merged**